tests/manage/z_cluster/test_drain_node_mon.py

```diff
@@ -0,0 +1,97 @@
+import logging
```

Comment:
We are doing rook-ceph operator restart during the mon failover, should we change the test name according to that
something like test_ rook_operator_restart_during_mon_failover ?

Reply:
done